remotespark/kernels/kernelmagics.py

```diff
@@ -70,10 +73,18 @@ def help(self, line, cell="", local_ns=None):
   </tr>
   <tr>
     <td>sql</td>
-    <td>%%sql -o tables<br/>SHOW TABLES</td> -q
-    <td>Executes a SQL query against the sqlContext. If the -o parameter is passed, the result of the query will be
-    available in the %%local Python context as a <a href="http://pandas.pydata.org/">Pandas</a> dataframe. If
-    the -q parameter is passed, the magic will return None instead of the dataframe.</td>
+    <td>%%sql -o tables -q<br/>SHOW TABLES</td>
+    <td>Executes a SQL query against the sqlContext.
+    Parameters:
+      <ul>
+        <li>-o VAR_NAME: The result of the query will be available in the %%local Python context as a
+          <a href="http://pandas.pydata.org/">Pandas</a> dataframe.</li>
+        <li>-q: The magic will return None instead of the dataframe (no visualization).</li>
```

Comment:
Can we expect users to understand that every cell has a return value? Can we explain that here?